src/main/java/org/opentripplanner/model/Frequency.java

```diff
@@ -93,14 +95,22 @@ public boolean equals(Object o) {
   }
 
   public String toString() {
-    return (
-      "<Frequency trip=" +
-      trip.getId() +
-      " start=" +
-      TimeUtils.timeToStrLong(startTime) +
-      " end=" +
-      TimeUtils.timeToStrLong(endTime) +
-      ">"
-    );
+    if (trip.getId().getFeedId().equals("agency2")) {
```

Comment:
Is this intentional?

Reply:
absolutely not, thanks. I was trying to debug why a test failed, and it turned out that it was sorting the Frequencies using the toString().